docs/prover/checking/coverage-info.md

```diff
@@ -0,0 +1,147 @@
+
+The right pane provides detailed information about individual lines and commands mapped to the lines group by the rule/method/invariant name (denoted *rule*) and the *value*, where each *value* refers to a particular `assign`, `assert` or `assume` TAC command that is mapped to the line. For instance, in the Tautology example, we can see that sub-command `other != manager` of line `11` matters (is in the unsat core), whereas `other != e.msg.sender` is not in the unsat core. Also, note that the pane shows e.g. both `other != manager` and `other == manager`; this is because in our TAC representation we encode `other != manager` as `!(other == manager)`. 
+
+
+### Visualization on TAC 
+On contrary to `.sol` and `.spec` visualization, we generate separate TAC unsat core visualization for individual rules/methods/invariants. To access it, click first on the particular rule/method/invariant in the `Rules` pane and then on the `Dump page` button as shown in Figure 3. 
+
+![](tac-visualization-button.png)
+Figure 3: The TAC visualization button.
+
+The visualization here consists only of 2 colors: `green` means that the command is needed and `red` means that the command is not needed (i.e. not in the unsat core). In particular, Figures 4a and 4b show the TAC visualization for the Tautology example. 
+
+![](tautology-tac-a-new.png)
+Figure 4a: The visualization of the Tautology example on TAC, part A. 
+
+![](tautology-tac-b-new.png)
+Figure 4b: The visualization of the Tautology example on TAC, part B.
+
+Namely, only the following TAC commands are in the unsat core:
+
+```
+→ require other != manager && other != e.msg.sender
+I95 = other287 SPEC
+I96 = manager269 SPEC
+tmp292293 = I95==I96 SPEC
+B97 = !tmp292293 SPEC
+...
+certoraAssume288289 = B97&&B102 SPEC
+assume certoraAssume288289
+...
+→ assert newManager != other || newManager != manager
+I158 = newManager419 SPEC
+I159 = other287 SPEC
+tmp441442 = I158==I159 SPEC
+B160 = !tmp441442 SPEC
+I161 = newManager419 SPEC
+I162 = manager269 SPEC
+tmp449450 = I161==I162 SPEC
+B163 = !tmp449450 SPEC
+certoraAssert437438 = B160||B163 SPEC
+assert certoraAssert437438,
+```
+
+Also, notice that some commands in the TAC dump are followed by "SPEC" or "SOL" with a violet background color; these are commands for which we have a mapping to `.spec` and `.sol` files, respectively. 
+
+## Basic vs. advanced mode
+
+The `--coverage_info` flag takes three possible values: `none`, `basic` and `advanced`:
+1. `none` means no unsat core analysis and no visualization, 
+2. `basic` means relatively fast bust possibly very imprecise analysis, 
```

Comment:
perhaps "analysis" ~> "unsat core information" (?)

Reply:
I somehow like `analysis` more than `information`. There is an implication, right? "imprecise analysis => imprecise unsat core information". The other direction of the implication does not hold (something between `data` and `information` might go wrong). 

But I will drop ` and no visualization` from `1.` so that it is more consistent.